src/mount.ts

```diff
@@ -269,33 +299,24 @@ export function mount(
         acc: { [key: string]: Function },
         [name, slot]: [string, Slot]
       ): { [key: string]: Function } => {
-        // case of an SFC getting passed
-        if (typeof slot === 'object' && 'render' in slot && slot.render) {
-          acc[name] = slot.render
-          return acc
-        }
+        if (Array.isArray(slot)) {
+          const normalized = slot.reduce<Array<Function | VNode>>(
+            (acc, curr) => {
+              const toF = slotToFunction(curr)
+              if (isObject(curr) && 'render' in curr) {
+                const rendered = h(toF as any)
```

Comment:
Not entirely clear why I need to pass `toF` (short for `toFunction`) into `h` here... but you need to.

Reply:
This is indeed weird. I don't know enough about slot internal behavior to help sadly.